webapp/packages/plugin-data-spreadsheet-new/src/theme/dark.module.scss

```diff
@@ -0,0 +1,13 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+@import '@cloudbeaver/core-theming/src/styles/theme-dark';
+
+:global .#{$theme-class} {
```

Comment:
We deleted these files on purpose, we dont want to use them anymore. We should define all styles in DataGridTable.module.css

Reply:
I rolled back changes, so we don't have a themed data grid. It is not required for now